### PR TITLE
Some profile UI

### PR DESCRIPTION
**Pull request by jacobburgo**
### What was the problem or new feature?

Made the page look nice. Can't get the onClick to work right, but I won't waste my time until we get some better functionality in the main Activities of the app.

### How does this PR fix the problem or implement the feature?

Makes things LOOK nice. Bout it.

### Checklist

- [ ] Coding style [Trailing whitespaces in files & no unneeded whitespaces] :sparkles:
- [ ] Are there any .idea/ files or .gradle/ files in this PR? :notebook:

### What issue(s) does this PR close?

Closes

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -19,6 +19,7 @@ android {
     }
 
     buildFeatures {
+        dataBinding = true
```
Comment (jacobburgo) [inline app/build.gradle:22]: Running the app cause all the data binding to break. This resolves that issue.

File: app/build.gradle [hunk 2]
```diff
@@ -35,6 +36,7 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    buildToolsVersion '33.0.0'
```
Comment (jacobburgo) [inline app/build.gradle]:
```suggestion
```

**Review by DerekKensinger — COMMENTED** [review]
Looks really cool to see all the coding you've been able to knock now that the Git is handled.

**Review by DerekKensinger — APPROVED** [review]
Okay I didn't select the approve option, just left selected Comment.  My bad.

**Pull request merged**